QA_PUSH_NOTIFICATION.md

```diff
@@ -26,250 +26,275 @@ It's a second custom NFT token to test.
-    1. Click on the `TTT - Test Token Test` item
+    1. [ ] Click on the `TTT - Test Token Test` item
         1. Nothing should happen
-    1. Click on the `TN1 - Test Nft` item
+    1. [ ] Click on the `TN1 - Test Nft` item
         1. Nothing should happen
-    1. Click on the `HTR - HATHOR` item
+    1. [ ] Click on the `HTR - HATHOR` item
         1. The **Balance** page should open
+
 1. **Send 4 tokens after turn on `Show amounts on notification` option**
-    1. Turn on the `push-notification` feature toggle
-    1. Go to the **Push Notification** page
-    1. Turn **on** the `Enable Push Notification` option
-    1. Turn **on** the `Show amounts on notification` option
-    1. Send HTR, TTT, TN1 and TNT to this wallet
+    1. [ ] Turn on the `push-notification` feature toggle
+    1. [ ] Go to the **Push Notification** page
+    1. [ ] Turn **on** the `Enable Push Notification` option
+    1. [ ] Turn **on** the `Show amounts on notification` option
+    1. [ ] Send HTR, TTT, TN1 and TNT to this wallet
         1. Wait until the notification arrives
         1. You should receive a notification of new transaction showing 2 amounts in the message
             > You have received 0.08 TNT, 0.05 TN1 and 2 other tokens on a new transaction.
-    1. Click on the notification
-    1. Wait until the modal with tx details open
+    1. [ ] Click on the notification
+    1. [ ] Wait until the modal with tx details open
         1. The `HTR - HATHOR` name should be in the primary color (purple)
         1. The `TTT - Test Token Test` name should be in gray
         1. The `TN1 - Test Nft` name should be in gray
         1. The `TNT - Test Nft Test` name should be in gray
-    1. Click on the `TTT - Test Token Test` item
+    1. [ ] Click on the `TTT - Test Token Test` item
         1. Nothing should happen
-    1. Click on the `TN1 - Test Nft` item
+    1. [ ] Click on the `TN1 - Test Nft` item
         1. Nothing should happen
-    1. Click on the `TNT - Test Nft Test` item
+    1. [ ] Click on the `TNT - Test Nft Test` item
         1. Nothing should happen
-    1. Click on the `HTR - HATHOR` item
+    1. [ ] Click on the `HTR - HATHOR` item
         1. The **Balance** page should open
+
 1. **Register `TTT` token and send 2 tokens after turn on `Show amounts on notification` option**
-    > WARNING: Not possible using `wallet-service` in `testnet` due to a validation that consults the fullnode.
-    > Jump to **Test open the wallet 2 weeks later** (19)
-    1. Register `TTT` token in the wallet
-    1. Turn on the `push-notification` feature toggle
-    1. Go to the **Push Notification** page
-    1. Turn **on** the `Enable Push Notification` option
-    1. Turn **on** the `Show amounts on notification` option
-    1. Send HTR and TTT to this wallet
+    >[!WARN]
+    >Not possible using `wallet-service` in `testnet` due to a validation that consults the fullnode.
+    >Jump to **Test open the wallet 2 weeks later** (19)
```

Comment:
question: Should we keep the step number here? We're using `1.` as a number for all steps precisely not to worry about renumbering them whenever we change some item.

Keeping a static reference here kind of defeats the purpose. Maybe just inform `Jump to the **[...]** step below`

```suggestion
    >Jump to **Test open the wallet 2 weeks later** (20)
```

Reply:
Done.